server/pxf-hdfs/src/test/java/org/greenplum/pxf/plugins/hdfs/ParquetWriteTest.java

```diff
@@ -842,8 +845,8 @@ public void testWriteChar() throws Exception {
     @Test
     public void testWriteNumeric() throws Exception {
         String path = temp + "/out/numeric/";
-        // precision is 38 and scale is 18
-        columnDescriptors.add(new ColumnDescriptor("dec1", DataType.NUMERIC.getOID(), 0, "numeric", new Integer[]{38, 18}));
+        int precision = 38, scale = 18;
```

Comment:
nit: we typically do not use this styling convention, is this something we want to keep or swap it to something like this: 
```suggestion
        int precision = 38;
        int scale = 18;